pkg/server/server.go

```diff
@@ -237,6 +244,15 @@ func New(
 
 		if v := tuning.Snapshot; v != nil {
 			logrus.WithFields(logrus.Fields{"threshold": v.Threshold, "trailing": v.Trailing}).Print("Configure dqlite raft snapshot parameters")
+			if v.Trailing < v.Threshold {
```

Comment:
I think this should come after the sanitization below. This helps catching the right behaviour.

I also think we should not error (and completely shut down the node) but instead log a warning and compute a suitable value instead. A rule of thumb might be `Threshold = Trailing * 3 / 4`. Also `Threshold = Trailing / 2` could work.